### PR TITLE
Update Location.pathname example

### PR DESCRIPTION
**Pull request by charliegroll**
### Description

The example on this page appears to reflect an old pathname for this doc (or at the very least doesn't reflect the current path).

### Motivation

If someone opens the console to try out `location.pathname` it might be reassuring for it to match the example on the page.

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
This is a bad example; it shows `HTMLAnchorElement#pathname`, not `Location#pathname`. Could you change it to the following instead?

```js
// Let's say we are on the URL https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname#examples
console.log(location.pathname); // "/en-US/docs/Web/API/Location/pathname"

File: files/en-us/web/api/location/pathname/index.md [hunk 1]
```diff
@@ -18,9 +18,8 @@ A string.
 ## Examples
 
 ```js
-// Let's say an <a id="myAnchor" href="/en-US/docs/Location.pathname"> element is in the document
-const anchor = document.getElementById("myAnchor");
-const result = anchor.pathname; // Returns:'/en-US/docs/Location.pathname'
+// Let's say we are on the URL https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname#examples
+console.log(location.pathname) // Returns:'/en-US/docs/Web/API/Location/pathname'
```
Comment (Josh-Cena) [inline files/en-us/web/api/location/pathname/index.md:22]:
```suggestion
console.log(location.pathname) // '/en-US/docs/Web/API/Location/pathname'
```

**Review by Josh-Cena — APPROVED** [review]
LGTM thanks

File: files/en-us/web/api/location/pathname/index.md [hunk 2]
```diff
@@ -18,9 +18,8 @@ A string.
 ## Examples
 
 ```js
-// Let's say an <a id="myAnchor" href="/en-US/docs/Location.pathname"> element is in the document
-const anchor = document.getElementById("myAnchor");
-const result = anchor.pathname; // Returns:'/en-US/docs/Location.pathname'
+// Let's say we are on the URL https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname#examples
+console.log(location.pathname) // '/en-US/docs/Web/API/Location/pathname'
```
Comment (Josh-Cena) [inline files/en-us/web/api/location/pathname/index.md:22]:
```suggestion
console.log(location.pathname); // '/en-US/docs/Web/API/Location/pathname'
```

**Comment by charliegroll** [comment]
Thanks for the reviews and cleanup at the end @Josh-Cena

**Pull request merged**